File: LeetCode/[E] - 136. Single Number/solution.go
package main

func singleNumber(nums []int) int {
	m := make(map[int]int, len(nums))
	for _, num := range nums {
		m[num] += 1
	}
	for k, v := range m {
		if v == 1 {
			return k
		}
	}
	return 0
}

File: LeetCode/[E] - 136. Single Number/solution.py
from typing import List


class Solution:
    def singleNumber(self, nums: List[int]) -> int:
        m = {}
        for num in nums:
            m[num] = 1 + m[num] if num in m else 1
        for key, value in m.items():
            if value == 1:
                return key
